File: FaceSample/app/src/main/java/com/regula/facesample/items/features/LivenessSessionIdItem.java
package com.regula.facesample.items.features;

import android.content.Context;

import com.regula.facesample.items.CategoryItem;
import com.regula.facesample.util.LivenessResponseUtil;
import com.regula.facesdk.FaceSDK;
import com.regula.facesdk.configuration.LivenessConfiguration;

import java.util.UUID;

public class LivenessSessionIdItem extends CategoryItem {

   @Override
   public void onItemSelected(Context context) {
      LivenessConfiguration configuration = new LivenessConfiguration.Builder()
              .setTag(UUID.randomUUID().toString())
              .build();
      FaceSDK.Instance().startLiveness(context, configuration, livenessResponse -> LivenessResponseUtil.response(context, livenessResponse));
   }

   @Override
   public String getDescription() {
      return "Set up session Id for liveness";
   }

   @Override
   public String getTitle() {
      return "Liveness Session Id";
   }
}

File: BasicSample/app/src/main/java/com/regula/facesamplekotlin/items/features/LivenessSessionIdItem.kt
package com.regula.facesamplekotlin.items.features

import android.content.Context
import com.regula.facesamplekotlin.category.CategoryItem
import com.regula.facesamplekotlin.util.LivenessResponseUtil
import com.regula.facesdk.FaceSDK
import com.regula.facesdk.configuration.LivenessConfiguration
import com.regula.facesdk.model.results.LivenessResponse
import java.util.*

class LivenessSessionIdItem : CategoryItem() {

    override fun onItemSelected(context: Context) {
        val configuration = LivenessConfiguration.Builder()
            .setTag(UUID.randomUUID().toString())
            .build()
        FaceSDK.Instance().startLiveness(
            context, configuration
        ) { livenessResponse: LivenessResponse -> LivenessResponseUtil.response(context, livenessResponse) }
    }

    override val title: String
        get() = "Liveness Session Tag"

    override val description: String
        get() = "Set up session tag for liveness"
}